.gitignore

```diff
@@ -1,3 +1,3 @@
 # Ignore generated Maven build files
 target/
-node_modules/
\ No newline at end of file
+node_modules/
```

Comment:
Is this a whitespace change or a newline? Adds noise to the edit history, I would remove

Reply:
Done